code-of-conduct.md

```diff
@@ -0,0 +1,54 @@
+# Code of Conduct
+
+This code of conduct outlines the expectations for participants within this community (primarily, but by no means exclusively, the educational mentoring sessions provided by RG for TP). This code of conduct is our way to encourage good behavior and discourage bad behavior in our sessions and the open source community. Additionally, it is worthwile to consider the importance of this document from an educational point of view, the potential effect a CoC has within the open source community, society at large and the positive development of mankind.
```

Comment:
Let's not mention any people specifically. In particular since this code of conduct is intended for use in the group project to come. Let's also not mention any sessions, but instead try and be all-encompassing.